pkgs/by-name/gh/ghostscript/package.nix

```diff
@@ -72,6 +73,12 @@ stdenv.mkDerivation rec {
   patches = [
     ./urw-font-files.patch
     ./doc-no-ref.diff
+
+    # Support SOURCE_DATE_EPOCH for reproducible builds
+    (fetchpatch2 {
+      url = "https://bugs.ghostscript.com/attachment.cgi?id=19241";
```

Comment:
I like getting the patch from Debian (the authors) better:

https://salsa.debian.org/debian/ghostscript/-/blob/debian/latest/debian/patches/2010_add_build_timestamp_setting.patch?ref_type=heads

I'm quite amenable to taking most of Debian's patches, as they all look sane from a quick scan.

Reply:
Thanks for the link, I only found https://bugs.debian.org/cgi-bin/bugreport.cgi?att=1;bug=794004;filename=ghostscript.diff.txt;msg=5, but was deterred by it also patching the changelog.